modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -125,6 +125,11 @@ export default {
     AposAdvisoryLockMixin,
     AposArchiveMixin
   ],
+  provide() {
+    return {
+      originalDoc: this.originalDoc
```

Comment:
To make it reactive, we must provide an object data property. As in your PR this requires an extra layer of nesting ("ref"), yes.